docs/guides/shortcuts.md

```diff
@@ -48,16 +43,27 @@ Bolt does many of the commonly required tasks for you. The steps you need to han
 * Build a reply message or do whatever you want to do
 * Call `ack()` as an acknowledgment
 
-Message actions request payloads have `response_url`, so that your app can reply to the action (even asynchronously after the acknowledgment). The URL is usable up to 5 times within 30 minutes of the action invocation. If you post a message using `response_url`, call `ctx.ack()` without arguments and use `ctx.respond()` to post a message.
+Message shortcut request payloads have `response_url`, so that your app can reply to the shortcut (even asynchronously after the acknowledgment). The URL is usable up to 5 times within 30 minutes of the shortcut invocation. If you post a message using `response_url`, call `ctx.ack()` without arguments and use `ctx.respond()` to post a message. Global shortcut request payloads don't have `response_url`.
```

Comment:
Might be help to mention `response_url_enabled` field for `conversation_select` elements. https://api.slack.com/reference/block-kit/block-elements#conversation_select. Using that in a modal is the recommended way to post a message.

Although if you don't feel like it fits into this section, that's completely fine.